Easydict/Swift/Service/Volcano/VolcanoService.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  VolcanoService.swift
+//  Easydict
+//
+//  Created by Jerry on 2024-08-11.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import Alamofire
+import Defaults
+import Foundation
+
+@objc(EZVolcanoService)
+public final class VolcanoService: QueryService {
+    // MARK: Public
+
+    override public func serviceType() -> ServiceType {
+        .volcano
+    }
+
+    override public func link() -> String? {
+        "https://translate.volcengine.com"
+    }
+
+    override public func name() -> String {
+        NSLocalizedString("volcano_translate", comment: "The name of Volcano Translate")
+    }
+
+    override public func supportLanguagesDictionary() -> MMOrderedDictionary<AnyObject, AnyObject> {
+        let orderedDict = MMOrderedDictionary<AnyObject, AnyObject>()
+        for (key, value) in VolcanoTranslateType.supportLanguagesDictionary {
+            orderedDict.setObject(value as NSString, forKey: key.rawValue as NSString)
+        }
+        return orderedDict
+    }
+
+    override public func ocr(_: EZQueryModel) async throws -> EZOCRResult {
```

Comment:
The `ocr` method usually does not need to be rewritten; its default implementation simply throws an error indicating that it is not supported.

Currently, we mainly use Apple's OCR, and if it fails, we attempt to use Youdao's OCR (if it is enabled).

Reply:
Removed in https://github.com/tisfeng/Easydict/pull/636/commits/6e1f4f03510bb055d204a35cc1f1b34eb8ac8ed8